var/spack/repos/builtin/packages/ruby-sqlite/package.py

```diff
@@ -0,0 +1,42 @@
+##############################################################################
+# Copyright (c) 2013-2016, Lawrence Livermore National Security, LLC.
+# Produced at the Lawrence Livermore National Laboratory.
+#
+# This file is part of Spack.
+# Created by Todd Gamblin, tgamblin@llnl.gov, All rights reserved.
+# LLNL-CODE-647188
+#
+# For details, see https://github.com/llnl/spack
+# Please also see the LICENSE file for our notice and the LGPL.
+#
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU Lesser General Public License (as
+# published by the Free Software Foundation) version 2.1, February 1999.
+#
+# This program is distributed in the hope that it will be useful, but
+# WITHOUT ANY WARRANTY; without even the IMPLIED WARRANTY OF
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the terms and
+# conditions of the GNU Lesser General Public License for more details.
+#
+# You should have received a copy of the GNU Lesser General Public
+# License along with this program; if not, write to the Free Software
+# Foundation, Inc., 59 Temple Place, Suite 330, Boston, MA 02111-1307 USA
+##############################################################################
+#
+
+from spack import *
+
+
+class RubySqlite(RubyPackage):
+    """This module allows Ruby programs to interface with the SQLite3 database
+    engine."""
+
+    homepage = "https://rubygems.org/gems/sqlite3"
+    url      = "https://rubygems.org/downloads/sqlite3-1.3.13.gem"
+
+    version('1.3.13', '96151574f04ca86a96b6ebe6dece9494', expand=False)
+
+    depends_on('sqlite')
+
+    def install(self, spec, prefix):
+        gem('install', '--local', 'sqlite3-{0}.gem'.format(self.version))
```

Comment:
Is there a reason that some format both `self.name` and `self.version` while others have hard-coded names?